pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -38,7 +38,8 @@
     "System.Threading.Tasks.Extensions": "4.0.0-rc3-24131-00",
     "System.Threading.Tasks.Parallel": "4.0.1-rc3-24131-00",
     "System.Threading.Thread": "4.0.0-rc3-24131-00",
-    "System.Threading.ThreadPool": "4.0.10-rc3-24131-00"
+    "System.Threading.ThreadPool": "4.0.10-rc3-24131-00",
+    "Libuv": "1.9.0-rc2-21014"
```

Comment:
@ellismg - this is how how/where we build it https://github.com/aspnet/Libuv-build. From this repo we build three separate libuv-build packages (Microsoft.AspNetCore.Internal.libuv-Darwin, Microsoft.AspNetCore.Internal.libuv-Linux and Microsoft.AspNetCore.Internal.libuv-Windows) which we then fold into the Libuv package in https://github.com/aspnet/libuv-package repo.